pom.xml

```
@@ -8,23 +8,22 @@
     <relativePath />
   </parent>
   <artifactId>gitlab-plugin</artifactId>
-  <version>${revision}${changelist}</version>
+  <version>1.7.1</version>
```

This allows the publication of incrementals so that we can easily test a pull request from an environment that is managing plugin versions as code.
```suggestion
  <version>${revision}${changelist}</version>
```